pkg/clusterresource/builder.go

```diff
@@ -133,6 +146,32 @@ func (o *Builder) Validate() error {
 		return fmt.Errorf("must set serving cert key to use with serving cert")
 	}
 
+	if len(o.HTTPProxy) > 0  || len(o.HTTPSProxy) > 0  {
+		if o.HTTPProxy == "" && o.HTTPSProxy == "" {
```

Comment:
Looks impossible for line 150 to trigger here as well. Was this meant to be a check that you don't try to set BOTH httpproxy and httpsproxy? If so I'm guessing these should both be != "".